src/crate/theme/rtd/crate/static/vendor/furo/styles/_scaffold.sass

```diff
@@ -0,0 +1,420 @@
+// This file contains the styles for the overall layouting of the documentation
+// skeleton, including the responsive changes as well as sidebar toggles.
+//
+// This is implemented as a mobile-last design, which isn't ideal, but it is
+// reasonably good-enough and I got pretty tired by the time I'd finished this
+// to move the rules around to fix this. Shouldn't take more than 3-4 hours,
+// if you know what you're doing tho.
+
+// HACK: Not all browsers account for the scrollbar width in media queries.
+// This results in horizontal scrollbars in the breakpoint where we go
+// from displaying everything to hiding the ToC. We accomodate for this by
+// adding a bit of padding to the TOC drawer, disabling the horizontal
+// scrollbar and allowing the scrollbars to cover the padding.
+// https://www.456bereastreet.com/archive/201301/media_query_width_and_vertical_scrollbars/
+
+// HACK: Always having the scrollbar visible, prevents certain browsers from
+// causing the content to stutter horizontally between taller-than-viewport and
+// not-taller-than-viewport pages.
+
+html
+  overflow-x: hidden
+  overflow-y: scroll
+  scroll-behavior: smooth
+
+.sidebar-scroll, .toc-scroll, article[role=main] *
+  // Override Firefox scrollbar style
+  scrollbar-width: thin
+  scrollbar-color: var(--color-foreground-border) transparent
+
+  // Override Chrome scrollbar styles
+  &::-webkit-scrollbar
+    width: 0.25rem
+    height: 0.25rem
+  &::-webkit-scrollbar-thumb
+    background-color: var(--color-foreground-border)
+    border-radius: 0.125rem
```

Comment:
That is leading to the scrollbars showing in Chrome
<img width="2095" alt="image" src="https://github.com/user-attachments/assets/3e226566-bf60-462a-9ade-2c1097015327">


They are not showing up in Safari
<img width="1624" alt="image" src="https://github.com/user-attachments/assets/b4f9c53d-25ec-4f7f-8cf9-ef912d3e702a">

Reply:
Thank you for investigating and reporting this. @msbt: Can you take care on behalf of your working branch/PR at GH-506?

Reply:
Fix already arrived per cdffe7a8d3f2f9072b7e? That was quick!

Reply:
@amotl @proddata I didn't have the issue in Firefox on Linux, could reproduce it with Chromium thought, is that better now?

Reply:
@proddata confirmed it is better now. Thank you.